plugins/doc_fragments/aci.py

```diff
@@ -104,6 +104,15 @@ class ModuleDocFragment(object):
     - The current object including default values will be unverifiable in a single task.
     type: bool
     aliases: [ no_verification, no_verify, suppress_verify ]
+  suppress_look_back:
```

Comment:
For me look back seems confusing. Perhaps it is my misinterpretation but for me it seems as if after transaction we would look again. Which seems to be more what the suppress_verification is doing. Not sure what is a better naming.

Reply:
Yeah I agree with you 100%, I was  a bit worried too. How does 'suppress_existing' or 'supress_previous' sounds to you? 
I am a little lost with the names too. 😆

Reply:
Probably `suppress_previous` makes more sense to me. @lhercot @shrsr @gmicol @anvitha-jain @sajagana what is your opinion?

Reply:
`suppress_previous` makes more sense to me among the given options.

Reply:
Yep sounds good, I'll change it now.